### DISCUSSION TITLE
Webcam take photo

### DISCUSSION THREAD

**Discussion by peterpranger** [cables-gl/cables_docs#527] [Answers & Questions] [answered]
Hello
I'm guite new to Cables. It takes a while to understand the logic.
Could anyone help do the following:

Use the webcam to take a photo and use the photo as a texture of new rectangle object (to show the photo). 
A mouse click (or button) should trigger the photo taking.

Really hope someone could help!
Cheers!
Peter

**Comment by steam0r** [comment, marked as answer]
sounds like a job for: 

https://cables.gl/op/Ops.Gl.InteractiveRectangle_v2

and:

https://cables.gl/op/Ops.Gl.DownloadTexture_v2

let me know if this works for your case!
